src/utils/TwitterSignatureValidator.test.ts

```diff
@@ -2,7 +2,7 @@ import { isValidTwitterSignature } from './TwitterSignatureValidator';
 
 describe('TwitterSignatureValidator', () => {
   it('should return true for valid signature', async () => {
-    const isValid = await isValidTwitterSignature({
+    const isValid = isValidTwitterSignature({
```

Comment:
@JohnnyJumper This is an async function as far as I know.

Reply:
It might used to be an async, but currently I don't see an async operations there. 

```typescript
export const isValidTwitterSignature = ({
  tokenId,
  owner,
  twitterHandle,
  validationSignature,
}: {
  tokenId: string;
  owner: string;
  twitterHandle: string;
  validationSignature: string;
}): boolean => {
  const tokenIdInDecimals = fromHexStringToDecimals(tokenId);
  const message = [
    tokenIdInDecimals,
    owner,
    standardKeys.twitter_username,
    twitterHandle,
  ]
    .map((value: string) => '0x' + sha3(value.startsWith('0x') ? hexToBytes(value) : value))
    .reduce((message, hashedValue) => message + hashedValue, '');
  const signerAddress = recover(message, validationSignature);
  return signerAddress === Cns.TwitterVerificationAddress;
};

```